app/Http/Controllers/User/AdminOrderInvoiceController.php

```diff
@@ -78,10 +86,39 @@ public function getClientInvoice($id)
                             style='color:white;'> </i>&nbsp;&nbsp;View</a>"
                                     ."   $editAction $action";
                         })
-                         ->rawColumns(['checkbox', 'date', 'invoice_no', 'total', 'paid', 'balance', 'status', 'action'])
+                         ->rawColumns(['checkbox', 'date', 'invoice_no', 'order_no', 'total', 'paid', 'balance', 'status', 'action'])
                         ->make(true);
     }
 
+    public static function getOrderLink($invoice)
+    {
+        $allInvoicesRelatedToOrder = $invoice->orderRelation()->pluck('order_id')->toArray();
```

Comment:
please use `->value('order_id')`. If value doesn't exist, it will give `null`. Using $allInvoicesRelatedToOrder[0] isn't required

Reply:
fixed